timeline.js

```diff
@@ -3,6 +3,24 @@ import { db } from './dbInstance.js'
 let hasLoaded = false
 
 class ReaderTimeline extends HTMLElement {
+  skip = 0
+  limit = 10
+  hasMoreItems = true
+  loadMoreBtn = null
+
+  constructor () {
+    super()
+    this.loadMoreBtn = document.createElement('button')
```

Comment:
Mind posting a screenshot of it? I can't seem to get it rendering on the latest release. Might be a CI issue?

Reply:
The site it live; can you try it now?

![Screenshot 2024-04-16 at 9 31 49 PM](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/d8a98a20-77f8-424c-85b7-d057ce2bc266)

Reply:
Looking good. I think the style might need to be made the same as the other buttons though?

Reply:
![Screenshot 2024-04-22 at 11 33 53 PM](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/30711a4c-0cfe-4771-b4fd-e99f1a4332a9)